mobify/progressive-web-scaffold
# Astro: Merlin's go to checkout opens cart modal

**JIRA**: https://mobify.atlassian.net/browse/HYB-983

## Changes
- created two different buttons for different behaviors
- if running astro, don't do anything with href, but open the modal

## How to test-drive this PR
- add item to cart
- press go to check out
- checkout modal should pop up
